Documentation/operations/upgrade.rst

```diff
@@ -144,15 +140,17 @@ version of Cilium are `here <https://github.com/cilium/cilium#stable-releases>`_
 Upgrading to the latest micro release ensures the most seamless experience if a
 rollback is required following the minor release upgrade.
 
-Step 2: Option A: Regenerate deployment files with upgrade compatibility (Recommended)
+Step 2: Use Helm to Upgrade your Cilium deployment
 --------------------------------------------------------------------------------------
 
-`Helm` can be used to generate the YAML files for deployment. This allows to
-regenerate all files from scratch for the new release. By specifying the option
-``--set config.upgradeCompatibility=1.7``, the generated files are guaranteed
-to not contain an options with side effects as you upgrade from version 1.7.
-You still need to ensure that you are specifying the same options as used for
-the initial deployment:
+`Helm` can be used to either upgrade Cilium directly or to generate a new set of
+YAML files that can be used to upgrade an existing deployment. This allows the
+flexibility to use Helm to manage Cilium directly or to apply the resulting file
+using the ``kubectl`` client to install Cilium in your cluster. By default, helm
```

Comment:
Nit:

```suggestion
using the ``kubectl`` client to install Cilium in your cluster. By default, Helm
```

Either we stick with `Helm` or `helm` or just "Helm".

Reply:
fixed this instance, but also standardized the file